tests/unit/config/test_review_config.py

```
@@ -0,0 +1,220 @@
+        domainz:
+          - api
+        languages:
+          - python
+        category: security
+""",
+        encoding="utf-8",
+    )
+    monkeypatch.chdir(tmp_path)
+    clear_config_cache()
+
+    with pytest.raises(ValueError, match="Unknown review.checklist.items keys"):
```

Ruff (0.15.20)

[warning] Pattern passed to `match=` contains metacharacters but is neither escaped nor raw

(RUF043)